helm/cloud-provider-vsphere/charts/cloud-provider-for-vsphere/values.yaml

```diff
@@ -0,0 +1,85 @@
+# Default values for vSphere CPI.
+# This is a YAML-formatted file.
+# vSphere CPI values are grouped by component
+
+global:
+  config:
+    enabled: false
+
+config:
+  enabled: true
+  name: vsphere-cloud-config
+  vcenter: "vcenter.local"
+  username: "user"
+  password: "pass"
```

Comment:
Should we remove user/password to enforce use of a secret like we do in other onprem providers?

Reply:
I plan to update this PR according to the discussion in slack regarding configuration interface. I forgot the discussion while opening this PR. Changing this PR as draft. 

Meanwhile you can give me an approval for this PR https://github.com/giantswarm/cloud-provider-vsphere-app/pull/30 :D

Reply:
done

Reply:
We decided to use this mechanism https://gigantic.slack.com/archives/C01F7T2MNRL/p1678703614073829

Also, we use this method for CAPVCD too. The only diff is we have only `refreshToken` there.